lib/macos.js

```diff
@@ -19,9 +19,29 @@ const parseMac = stdout => {
 	}
 };
 
-module.exports = async () => {
-	const {stdout} = await execFile(bin);
+const getArguments = options => {
+	if (!options) {
+		return [];
+	}
+
+	const args = [];
+	if (options.accessCheck === false) {
+		args.push('--no-access-check');
+	}
+
+	if (options.appleScript === false) {
```

Comment:
I think this option should be named after what it does, not that it uses AppleScript.

Reply:
for `appleScript` as for `accessCheck` I was trying to generalise which kind of accessibility dialogs we want to avoid being triggered. No problem in changing it though. What about?

 - `accessibilityCheck`
 - `screenRecordingCheck`
 - `url`

Reply:
👍